diff --git gresid.F gresid.F
index 24929e0..8e4feb4 100644
--- gresid.F
+++ gresid.F
@@ -44,6 +44,7 @@
 *    Boston, MA  02111-1307  USA
 *-
 
+      USE IFPORT
       IMPLICIT NONE
 
       REAL S
@@ -51,14 +52,6 @@
       REAL X,Y,R,W,GNEXT,G
       LOGICAL FTF,FIRST
 
-#if HAVE_RAND
-      REAL RAND
-#elif HAVE_RANDOM
-      REAL RANDOM
-#else
- error "Can't find random-number function"
-#endif
-
       SAVE GNEXT,FTF,FIRST
       DATA FTF,FIRST / .TRUE.,.TRUE. /
 
diff --git random.F random.F
index 0d5bd74..4fe9606 100644
--- random.F
+++ random.F
@@ -1,5 +1,5 @@
 #include <sla_config.h>
-      REAL FUNCTION sla_RANDOM (SEED)
+      REAL FUNCTION sla_RANDOM (UNUSED_SEED)
 *+
 *     - - - - - - -
 *      R A N D O M
@@ -17,7 +17,7 @@
 *  1)  The result is a pseudo-random REAL number in the range
 *      0 <= sla_RANDOM < 1.
 *
-*  2)  SEED is used first time through only.
+*  2)  SEED is not used, but is left in as legacy.
 *
 *  Called:  RAN or RAND (a REAL function returning a random variate --
 *           the precise function which is called depends on which functions
@@ -43,45 +43,11 @@
 *    Free Software Foundation, Inc., 59 Temple Place, Suite 330, 
 *    Boston, MA  02111-1307  USA
 *-
-
+      USE IFPORT
       IMPLICIT NONE
-
-      REAL SEED
-
-#if HAVE_RAND
-      REAL RAND
-#elif HAVE_RANDOM
-      REAL RANDOM
-#else
- error "Can't find random-number function"
-#endif
-
-      REAL AS
-      INTEGER ISEED
-      LOGICAL FIRST
-      SAVE FIRST
-      DATA FIRST /.TRUE./
-
-
-
-*  If first time, turn SEED into a large, odd integer
-      IF (FIRST) THEN
-         AS=ABS(SEED)+1.0
-         ISEED=NINT(AS/10.0**(NINT(ALOG10(AS))-6))
-         IF (MOD(ISEED,2).EQ.0) ISEED=ISEED+1
-         FIRST=.FALSE.
-#if HAVE_RAND
-         AS = RAND(ISEED)
-#endif
-      ELSE
-         ISEED=0
-      END IF
+      REAL UNUSED_SEED
 
 *  Next pseudo-random number
-#if HAVE_RAND
       sla_RANDOM=RAND(0)
-#elif HAVE_RANDOM
-      sla_RANDOM=RANDOM(ISEED)
-#endif
 
       END
diff --git wait.f wait.f
index 94ed70f..b5693da 100644
--- wait.f
+++ wait.f
@@ -39,11 +39,11 @@
 *    Boston, MA  02111-1307  USA
 *
 *-
-
+      USE IFPORT
       IMPLICIT NONE
 
       REAL DELAY
 
-      CALL SLEEP(NINT(DELAY))
+      CALL SLEEPQQ(NINT(DELAY*1000))
 
       END
