diff --git a/conda/_vendor/boltons/setutils.py b/conda/_vendor/boltons/setutils.py
index 6f7cc5be7..002548d0a 100644
--- a/conda/_vendor/boltons/setutils.py
+++ b/conda/_vendor/boltons/setutils.py
@@ -17,7 +17,10 @@ from __future__ import print_function
 
 from bisect import bisect_left
 from itertools import chain, islice
-from collections import MutableSet
+try:
+    from collections.abc import MutableSet
+except ImportError:
+    from collections import MutableSet
 import operator
 
 try:
